decidim-system/app/packs/src/decidim/system/api_users_handler.js

```diff
@@ -0,0 +1,12 @@
+import PasswordToggler from "src/decidim/password_toggler";
+
+const initializeApiSecretToggler = () => {
+  const apiUserPasswords = document.querySelectorAll(".api-user-secret");
+  apiUserPasswords.forEach((userPassword) => {
+    new PasswordToggler(userPassword).init();
```

Comment:
It seems there is some issue with this, as the button is incorrectly aligned:
![Misaligned show button](https://github.com/user-attachments/assets/45869d5a-fb6a-451e-92c0-1813c03ec621)

Also, I get the following JS error on the page, could be related:
```
Uncaught TypeError: Cannot read properties of null (reading 'addEventListener')
    at PasswordToggler.init (password_toggler.js:24:1)
    at api_users_handler.js:5:1
    at NodeList.forEach (<anonymous>)
    at initializeApiSecretToggler (api_users_handler.js:4:1)
    at HTMLDocument.<anonymous> (api_users_handler.js:9:1)
```